src/components/Feed/SchedulingFeedPreview/SchedulingFeedPreview.styled.ts

```
@@ -0,0 +1,103 @@
+	}
+`;
+
+export const ScheduleGrid = styled.div`
+	position: relative;
+	min-height: 0;
+	flex: 1 1 auto;
+	margin-left: ${SCHEDULING_TIME_COLUMN_WIDTH}px;
+	border-top: 1px solid ${theme.color.border.tertiary};
+	border-left: 1px solid ${theme.color.border.tertiary};
+	background-color: ${theme.color.bg.primary};
+	background-image: ${PREVIEW_BLOCKS_BACKGROUND},
```

ESLint

[error] Replace `·` with `⏎↹↹`

(prettier/prettier)